charts/mission-control/templates/scraper.yaml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flanksource/duty/main/schema/openapi/scrape_config.schema.json
+apiVersion: configs.flanksource.com/v1
+kind: ScrapeConfig
+metadata:
+  name: mission-control-scraper
+spec:
+  sql:
+    # Agent Scraper
+    - connection: connection://{{.Values.connectionName}}
+      id: '$.id'
+      name: '$.name'
+      type: MissionControl::Agent
+      class: Agent
+      description: '$.description'
+      status: '$.status'
+      health: '$.health'
+      createFields: ['$.created_at']
+      deleteFields: ['$.deleted_at']
```

arent these the defaults ? 
```suggestion
      createFields: ['$.created_at']
      deleteFields: ['$.deleted_at']
```

---

Couldn't find any defaults for this in the codebase

---

We have this as default for change mapping